### ISSUE TITLE
RC Test: gravity-force-lab-basics:1.0.0-rc.2

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#438]
@terracoda, @emily-phet, @jessegreenberg, gravity-force-lab-basics:1.0.0-rc.2 is ready for RC testing. This is the second RC test for this sim (see https://github.com/phetsims/QA/issues/429). This is being published for the first time, and includes sound as well as Interactive Descriptions. Document issues in https://github.com/phetsims/gravity-force-lab-basics/issues and link to this issue. There are many changes from the previous version, and a few were difficult to copy from master, so I would recommend testing a bit more thoroughly than just a a spot check here. Especially as it pertains to the descriptions.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test.
- When making an issue, check to see if it was in a previously published version
- If there is a console available, check for errors and include them in the Problem Description.

<h3>Issues to Verify</h3>

Instead of just issues, these are links directly to the comment which should have the instructions about what to verify:

- [x] [Should get "closest" alert when closest with a constant radius](https://github.com/phetsims/gravity-force-lab-basics/issues/183#issuecomment-541943443)
- [x] [Need to double check that Size of Arrows and Effort of Robots (cancel out)](https://github.com/phetsims/gravity-force-lab-basics/issues/182#issuecomment-541900986)
- [x] [Investigate best min and max values for mass sliders](https://github.com/phetsims/gravity-force-lab-basics/issues/181#issuecomment-541944691)
- [x] [iPadOS 13 Checkboxes won't select/deselect with VO](https://github.com/phetsims/gravity-force-lab-basics/issues/176) but the general issue is [here](https://github.com/phetsims/scenery/issues/1001) I don't think there is any difference between the two, but feel free to close both if helpful when checking. 

This sim was also not included in some of the batch maintenance releases that recently went out, so please make sure these are fixed in this rc as well:
- [x] [Depressed buttons on iOS 13 look wrong](https://github.com/phetsims/sun/issues/526)
- [x] [iOS 13 Full Screen enabled](https://github.com/phetsims/phet-core/issues/64)

These issues should have the "status:ready-for-qa" label. Check these issues off and close them if they are fixed. Otherwise, post a comment in the issue saying that it wasn't fixed and link back to this issue. If the label is "status:ready-for-review" or "status:fixed-pending-deploy" then assign back to the developer when done, even if fixed.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.2/phet/gravity-force-lab-basics_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.2/phet/gravity-force-lab-basics_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.2/phet/xhtml/gravity-force-lab-basics_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/14SmRPdAdphYfSf-7n0gR2iltcZUHmnQqsmGKd8hTBYg/edit#gid=491633625)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.2/phet/gravity-force-lab-basics_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.0.0-rc.2/phet/gravity-force-lab-basics_all_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/14SmRPdAdphYfSf-7n0gR2iltcZUHmnQqsmGKd8hTBYg/edit#gid=491633625)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver (only if specified in testing issue)

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<h3>Final Requests</h3>

- [x] If this sim is in [this list](https://github.com/phetsims/perennial/blob/master/data/accessibility), make an issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by lmulhall-phet** [comment]
Making top since we don't have anything else in the repo right now. (Except for issues that languishing.)

**Comment by KatieWoe** [comment]
@jonathanolson mentioned it, but https://github.com/phetsims/QA/issues/432#issuecomment-542567431 needs to be implemented here.

**Comment by ariel-phet** [comment]
bumping down to high priority, since #432 is for fixes to broken sims that are "live" in the world.

**Comment by KatieWoe** [comment]
https://github.com/phetsims/QA/issues/432#issuecomment-542567431 was not applied.  That means that https://github.com/phetsims/phet-core/issues/64 failed I believe

**Comment by KatieWoe** [comment]
QA is done

**Comment by zepumph** [comment]
Thanks! We will proceed with a spot check rc over in https://github.com/phetsims/gravity-force-lab-basics/issues/184#issuecomment-542814736.
